jenkins_pipelines/environments/manager-5.0-qe-build-validation-NUE

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env groovy
+
+node('sumaform-cucumber') {
+    properties([
```

Comment:
You are missing the `def minionList = `, the pipeline will not run the minions that are not declared in that list.

Reply:
Thank you, Maxime. I used the wrong file to start as a template. I will fix it soon.

Reply:
Fixed.